index.html

```diff
@@ -22,31 +22,34 @@
   <body>
     <div class="page-container">
       <div class="header1">
-      <header class="headercontainer">
-        <a href="#" class="logo">Megha</a>
-        <nav><i class="fa-solid fa-bars hambrgricon"></i></nav>
-      </header>
-    </div>
-<div class="header2">
-      <header class="headercontainer">
-        <a href="#" class="logo">Megha</a>
-        <nav><ul class="navlist">
-          <li> <a href="#">Portfolio</a></li>
-          <li> <a href="#">About</a></li>
-          <li> <a href="#">Contact</a></li>
-        </ul></nav>
-      </header>
-    </div>
+        <header class="headercontainer">
+          <a href="#" class="logo">Megha</a>
+          <nav><i class="fa-solid fa-bars hambrgricon" id="hmbg"></i></nav>
+        </header>
+      </div>
+      <div class="header2">
+        <header class="headercontainer">
+          <a href="#" class="logo">Megha</a>
+          <nav>
+            <ul class="navlist">
+              <li class="linkitems"><a href="#portfolio">Portfolio</a></li>
+              <li class="linkitems"><a href="#about">About</a></li>
+              <li class="linkitems"><a href="#contact">Contact</a></li>
+            </ul>
+            <i class="fa-solid fa-times closeicon" id="closebtn"></i>
+          </nav>
```

Comment:
When I click on any link in the desktop navbar, though it displays the correct part of the page, however, it also makes the navbar disappear. The navbar in the desktop view should always be visible. Consider fixing it.